lib/api/projects.rb

```diff
@@ -28,6 +28,61 @@ class Projects < Grape::API
         end
       end
 
+      # Retrieve all jobs for a project
+      #
+      # Parameters
+      #   id (required) - The ID of a project
+      # Example Request
+      #   GET /projects/:id/jobs
+      get ":id/jobs" do
+        project = Project.find(params[:id])
+
+        if project.present? && current_user.can_access_project?(project.gitlab_id)
+          project.jobs
+        end
+      end
+
+      # Register new job for project
+      #
+      # Parameters
+      #   id (required) - The ID of a project
+      #   name (required) - The job name
+      #   commands (required) - The command line script for the job
+      # Example Request
+      #   POST /projects/:id/jobs
+      post ":id/jobs" do
+        required_attributes! [:name, :commands]
+
+        project = Project.find(params[:id])
+
+        if project.present? && current_user.can_access_project?(project.gitlab_id)
+          job = project.jobs.new({name: params[:name], commands: params[:commands]})
```

Comment:
Line is too long. [84/80]<br>Redundant curly braces around a hash parameter.<br>Space inside { missing.<br>Space inside } missing.